pip-wheel-metadata/qiskit_terra.dist-info/LICENSE.txt

```diff
@@ -0,0 +1 @@
+
```

Comment:
did you commit this file by mistake?

Reply:
Yep...

Ignis has a random clifford function based on Smolin Koenig paper.